docs/narr/introduction.rst

```diff
@@ -353,38 +374,46 @@ You can return a Python dictionary:
     def myview(request):
         return {'a':1}
 
-When this view callable is called by Pyramid, the ``{'a':1}`` dictionary will
-be rendered to a response on your behalf.  The string passed as ``renderer=``
-above is an :term:`asset specification`.  It is in the form
-``packagename:directoryname/filename.ext``.  In this case, it refers to the
-``mytemplate.pt`` file in the ``templates`` directory within the ``myapp``
-Python package.  Asset specifications are omnipresent in Pyramid. See
-:ref:`intro_asset_specs` for more information.
+By configuring your view to use a renderer, you tell Pyramid to use the
+``{'a':1}`` dictionary and the specified template to render a response on your
+behalf.
+
+The string passed as ``renderer=`` above is an :term:`asset specification`.
+Asset specifications are widely used in Pyramid. They allow for more reliable
+customization. See :ref:`intro_asset_specs` for more information.
 
 Example: :ref:`renderers_chapter`.
 
-Event system
-~~~~~~~~~~~~
+Use events to coordinate
```

Comment:
"Use events to coordinate actions". "actions" seems to fit, but there may be a better word or term.